CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# dbt_hubspot v0.19.1
+
```

Comment:
Let's add this to an under the hood section and include the PR link for this change.